tests/AppBundle/Controller/Admin/BlogControllerTest.php

```diff
@@ -70,7 +70,7 @@ public function testAdminBackendHomePage()
         $client = $this->getAdminClient();
 
         $crawler = $client->request('GET', '/en/admin/post/');
-        $this->assertSame(Response::HTTP_OK, $client->getResponse()->getStatusCode());
+        $this->assertTrue($client->getResponse()->isSuccessful());
```

Comment:
... however, here the `isSuccessful()` method is too liberal:

```php
    public function isSuccessful()
    {
        return $this->statusCode >= 200 && $this->statusCode < 300;
    }
```

I agree with @bocharsky-bw and I think we should revert this change and go back to `Response::HTTP_OK`